addons/test_mass_mailing/tests/test_mailing.py

```diff
@@ -378,3 +378,33 @@ def test_mailing_mailing_list_optout(self):
             check_mail=True
         )
         self.assertEqual(mailing.ignored, 2)
+
+    @users('user_marketing')
+    def test_mass_mailing_duplicates(self):
+        """
+        Tests whether function `_get_seen_list` is correctly able to identify duplicate emails,
+        even through different batches.
+        Mails use different names to make sure they are recognized as duplicates even without being
+        normalized (e.g.: '"jc" <0@example.com>' and '"vd" <0@example.com>' are duplciates)
+        """
+        BATCH_SIZE = 5
+        names = ['jc', 'vd']
+        emails = [f'{i}@example.com' for i in range(BATCH_SIZE)]
```

Comment:
Just to have a bioutifoul naming ?
```suggestion
        emails = [f'test.{i}@example.com' for i in range(BATCH_SIZE)]
```